tests/devapps/NetCoreTestApp/Program.cs

```diff
@@ -42,30 +43,56 @@ public class Program
         private readonly static string ClientIdForConfidentialApp = "<enter id>";
 
         private readonly static string Username = ""; // used for WIA and U/P, cannot be empty on .net core
-        private readonly static string Authority = "https://login.microsoftonline.com/organizations/v2.0"; // common will not work for WIA and U/P but it is a good test case
+        private readonly static string Authority = "https://login.microsoftonline.com/organizations"; // common will not work for WIA and U/P but it is a good test case
         private readonly static IEnumerable<string> Scopes = new[] { "user.read" }; // used for WIA and U/P, can be empty
 
         private const string GraphAPIEndpoint = "https://graph.microsoft.com/v1.0/me";
 
         public static void Main(string[] args)
         {
+            // with discrete logging calls (current API)
+            IPublicClientApplication pca = PublicClientApplicationBuilder
+                .Create(ClientIdForPublicApp)
+                .WithAadAuthority(AadAuthorityAudience.AzureAdMultipleOrgs)
+                .WithLoggingCallback(Log)
```

Comment:
>.WithLogging [](start = 16, length = 12)

.WithLogging(Log, LogLevel, EnablePii) ?

Reply:
this is simple, and we can provide default values.